File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/issuedTicket/exception/IssuedTicketNotFoundException.java
package band.gosrock.domain.domains.issuedTicket.exception;


import band.gosrock.common.exception.DuDoongCodeException;
import band.gosrock.common.exception.GlobalErrorCode;

public class IssuedTicketNotFoundException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new IssuedTicketNotFoundException();

    private IssuedTicketNotFoundException() {
        super(GlobalErrorCode.ISSUED_TICKET_NOT_FOUND);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/issuedTicket/exception/IssuedTicketNotFoundException.kt
package band.gosrock.domain.domains.issuedTicket.exception

import band.gosrock.common.exception.DuDoongCodeException

class IssuedTicketNotFoundException private constructor() : DuDoongCodeException(IssuedTicketErrorCode.ISSUED_TICKET_NOT_FOUND) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = IssuedTicketNotFoundException()
    }
}
